server/bin/pbench-dispatch.sh

```diff
@@ -46,72 +46,63 @@ test -d "${install_dir}" || doexit "Bad install_dir=${install_dir}"
 backup_dir=$(getconf.py pbench-backup-dir pbench-server)
 test -z "${backup_dir}" -o -d "${backup_dir}" || doexit "Bad backup_dir=${backup_dir}"
 
-errlog=${LOGSDIR}/${PROG}/${PROG}.error
+log_init ${PROG}
+
 mkdir -p ${LOGSDIR}/${PROG}
 sts=${?}
 if [[ ${sts} != 0 ]]; then
-    echo "Failed: \"mkdir -p ${LOGSDIR}/${PROG}\", status ${sts}" >> ${errlog}
-    exit 1
+    log_exit "${TS}: Failed: \"mkdir -p ${LOGSDIR}/${PROG}\", status ${sts}"
```

Comment:
You're not specifying a second argument here (like you do in all the other changed lines which invoke `log_exit`).

Reply:
The `log_exit` function takes 2 arguments, but the 2nd is optional (allows the caller to specify an exit status other than `1`.